doc/source/user_guide/10min.rst

```diff
@@ -178,12 +178,26 @@ Getitem (``[]``)
 ~~~~~~~~~~~~~~~~
 
 For a :class:`DataFrame`, passing a single label selects a column and
-yields a :class:`Series` equivalent to ``df.A``:
+yields a :class:`Series`:
 
 .. ipython:: python
 
    df["A"]
 
+If the label only contains letters, numbers, and underscores, you can
```

Comment:
Mind confirming I have this right?

Reply:
pandas allows strings that pass `isidentifier` https://docs.python.org/3/library/stdtypes.html#str.isidentifier